lib/rbac/filterer.rb

```diff
@@ -563,20 +561,27 @@ def to_class(klass)
       klass.kind_of?(String) || klass.kind_of?(Symbol) ? klass.to_s.constantize : klass
     end
 
-    def apply_scope(klass, scope)
-      klass = klass.all
-      scope_name = Array.wrap(scope).first
+    def send_scope(klass, scope)
+      scope_name = Array(scope).first
       if scope_name.nil?
         klass
       elsif klass.nil? || !klass.respond_to?(scope_name)
         class_name = klass.nil? ? "Object" : klass.name
-        raise _("Named scope '%{scope_name}' is not defined for class '%{class_name}'") % {:scope_name => scope_name,
-                                                                                           :class_name => class_name}
+        raise _("Named scope '#{scope_name}' is not defined for class '#{class_name}'")
```

Comment:
this removes the ability to localize.
Again, why was this line changed?

Reply:
make sense, reverting